reference/CloudPay-API-Specification.yaml

```diff
@@ -1381,6 +1381,143 @@ paths:
 
         **Note** *Only one of the four authorisation parameters are required*
     parameters: []
+  /api/v1/account/transactions:
+    get:
+      operationId: get-api-v1-account-transactions
+      summary: Retrieve the authenticated customer's transactions
+      tags:
+        - Account
+        - Transactions
+      parameters:
+        - $ref: '#/components/parameters/apiJwtTokenParam'
+        - $ref: '#/components/parameters/apiSessionIdParam'
+        - $ref: '#/components/parameters/thirdPartyAuthCookie'
+        - $ref: '#/components/parameters/authCookie'
+        - in: query
+          name: page
+          schema:
+            type: integer
+            minimum: 1
+            default: 1
+          description: The current page
+        - in: query
+          name: size
+          schema:
+            type: integer
+            minimum: 1
+            default: 100
+          description: The page size per page
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                description: 'Customer transactions list ordered by createdat DESC'
```

Comment:
Should we provide more natural description, something like 'Customer transactions listed in descending order of creation timestamp' than code-like `ordered by createdat DESC`

Reply:
I think we can add sort order and set default DESC What do you think?

Reply:
Sounds like a good idea, but I'm not aware if that's a requirement, we can check with the team I suppose. My comment was only about the description :)